internal/pkg/fetch/fetch.go

```diff
@@ -148,3 +148,7 @@ func getRefFile(parentFile *os.File) (*os.File, error) {
 
 	return refOutput, nil
 }
+
+func ExtRefDocument(sbomURL string, opts *options.FetchOptions) (*sbom.Document, error) {
+	return document(sbomURL, opts)
```

Comment:
Can we just put the `document()` logic in here directly if this is intended to be called from other packages?